helm/cas-provision/templates/job-runner/jobRunnerRole.yaml

```diff
@@ -0,0 +1,45 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-job-runner
+  namespace: {{ .Release.Namespace }}
```

Comment:
I don't think that we need to provide the `namespace` metadata? I would assume that's added by OpenShift when creating the object?